src/script/components/calling/CallingCell/CallIngParticipantList/CallingParticipantList.tsx

```diff
@@ -78,30 +82,56 @@ export const CallingParticipantList = ({
     showContextMenu({event, entries, identifier: 'participant-moderator-menu'});
   };
 
+  const participantsList = participants
+    .slice()
+    .sort((participantA, participantB) => sortUsersByPriority(participantA.user, participantB.user));
```

Comment:
I would memorize it, sorting could be an expensive computation, and as long as we have many re-renders, it would be great not to calculate such a thing multiple times

Reply:
done in https://github.com/wireapp/wire-webapp/pull/18250/commits/bcec62eb5a9d55bf4a0af0d7d0e925e4d1c1433f